File: iniciante/javascript/1038.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let entrada = lines.shift().split(' ')
let valores = [0.0, 4.00, 4.50, 5.00, 2.00, 1.50]

console.log(`Total: R$ ${(valores[parseInt(entrada[0])] * entrada[1]).toFixed(2)}`)

File: iniciante/python/1038.py
# -*- coding: utf-8 -*-

entrada = input().split(' ')
lista_preco = [4.00, 4.50, 5.00, 2.00, 1.50]
total = lista_preco[int(entrada[0]) - 1] * int(entrada[1])
print(f'Total: R$ {total:.2f}')
